From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Thu, 2 Aug 2018 16:38:09 +0200
Subject: org.gnome.Shell.desktop.in.in: enable JS backtrace on crashes

As per other included patches, this could be done using a mode too, but
since we're in the transition to yaru, better to just enable for everyone,
and once we have just one ubuntu mode we can define this in the .json only.

Forwarded: not-needed
---
 data/org.gnome.Shell.desktop.in.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/org.gnome.Shell.desktop.in.in b/data/org.gnome.Shell.desktop.in.in
index 09543e8..fa5cde7 100644
--- a/data/org.gnome.Shell.desktop.in.in
+++ b/data/org.gnome.Shell.desktop.in.in
@@ -2,7 +2,7 @@
 Type=Application
 Name=GNOME Shell
 Comment=Window management and application launching
-Exec=@bindir@/gnome-shell
+Exec=env SHELL_DEBUG=backtrace-crashes-all @bindir@/gnome-shell
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-shell
 X-GNOME-Bugzilla-Component=general
